package.json

```diff
@@ -24,28 +25,34 @@
         "prepublish": "yarn run build",
         "lint": "yarn run eslint src/*.ts",
         "lint:fix": "yarn run eslint --fix src/*.ts",
-        "build": "yarn run rimraf dist && yarn run lint && yarn run tsc && yarn run copyfiles -au1 src/sureql.sh src/resources/**/* dist/"
+        "build": "yarn run rimraf dist && yarn run lint && yarn run tsc && yarn run copyfiles -au1 src/resources/**/* dist/src/ && yarn run copyfiles -au1 test/queries/**/* test/specs/**/* dist/test/",
```

Comment:
In `package.json` scripts, you only need to prefix with `yarn run` when calling a named script, it's not needed when calling a bin from a dependency. So this can be:

```json
"build": "rimraf dist && yarn run lint && tsc && copyfiles -au1 src/resources/**/* dist/src/ && copyfiles -au1 test/queries/**/* test/specs/**/* dist/test/",
```